packages/capabilities/test/capabilities/access.test.js

```diff
@@ -371,3 +371,290 @@ describe('access capabilities', function () {
     })
   })
 })
+
+/**
+ * @param {Ucanto.Signer<Ucanto.DID<'key'>>} alice
+ * @param {Ucanto.Principal<Ucanto.DID<'key'>>} service
+ */
+function createSelfIssuedDelegateInvocation(alice, service) {
+  return Access.delegate
+    .invoke({
+      issuer: alice,
+      audience: service,
+      with: alice.did(),
+      nb: {
+        delegations: {},
+      },
+    })
```

Comment:
> This function is used just once and does not really improve readability of test.

I removed it https://github.com/web3-storage/w3protocol/pull/420/commits/57f51ea034b7134d7593481445a4f2b8650f3d31

> I would suggest changing it as follows and reusing it all the other tests

I'd prefer not to wrap use of `Access.delegate.invoke()` by other tests unnecessarily, because I find it easier to reason about the other tests if they just use the capability parser directly vs using it through a wrapper like `delegateAccess`